Makefile

```diff
@@ -21,6 +21,7 @@ travis-build: fetch-dependencies
 	find caddy/caddymain -name 'run.go' -type f -exec sed -i -e "s/\/\/ This is where other plugins get plugged in (imported)/_ \"github.com\/txtdirect\/txtdirect\/caddy\"/g" -- {} + && \
 	find caddy/caddymain -name 'run.go' -type f -exec sed -i -e '/_ "github.com\/txtdirect\/txtdirect\/caddy"/a _ "github.com\/miekg\/caddy-prometheus"' -- {} + && \
 	find caddy/caddymain -name 'run.go' -type f -exec sed -i -e '/_ "github.com\/txtdirect\/txtdirect\/caddy"/a _ "github.com\/captncraig\/caddy-realip"' -- {} + && \
+	find caddy-copy/caddy/caddymain -name 'run.go' -type f -exec sed -i -e 's/const enableTelemetry = true/const enableTelemetry = false/g' -- {} +
```

Comment:
```diff
-find caddy-copy/caddy/...
+find caddy/caddy/...
```

Reply:
The build still fails - should line 24 of the makefile be changed to this?

```
find caddy/caddymain...
```

Reply:
Sorry, just saw your comment 😅

Reply:
No problem - glad we came to the same conclusion independently :smile: